docs/usage/setup-azure-devops.md

```diff
@@ -79,6 +79,32 @@ module.exports = {
 For the `endpoint` key, replace `YOUR-ORG` with your Azure DevOps organization.
 For the `repositories` key, replace `YOUR-PROJECT/YOUR-REPO` with your Azure DevOps project and repository.
 
+### Yarn users
+
+To do a succesfull `yarn install` you need to match the URL of the registry fully.
+Use the `baseUrl` config option to specify the full path to the registry.
+
+```javascript
+module.exports = {
+  platform: 'azure',
+  endpoint: 'https://<redacted>.visualstudio.com/',
+  token: process.env.TOKEN,
+  hostRules: [
+    {
+      baseUrl:
+        'https://<redacted>.pkgs.visualstudio.com/_packaging/<redacted>/npm/registry/',
+      token: process.env.TOKEN,
+      hostType: 'npm',
+    },
+    {
+      domainName: 'github.com',
+      token: process.env.GITHUB_COM_TOKEN,
+    },
+  ],
+  repositories: ['YOUR-PROJECT/YOUR-REPO'],
+};
+```
+
```

Comment:
```suggestion
repo **.npmrc**
    ```ini
    registry=https://<redacted>.pkgs.visualstudio.com/_packaging/<redacted>/npm/registry/
    always-auth=true
    ```
```

Don't know how to sucessfully escape the suggestion, but we should also add the required `.npmrc` settings

Reply:
I've copy/pasted your example into the docs. 😉